spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/FeignClientFactoryBean.java

```diff
@@ -39,12 +39,15 @@
 import feign.codec.Decoder;
 import feign.codec.Encoder;
 import feign.codec.ErrorDecoder;
+import feign.hystrix.HystrixFeign;
```

Comment:
Another option I was looking at would be to put it in `HystrixTargeter`, so the check to see if it's a HystrixBuilder is already done